api/v1/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
Did just your local build require this update? The downstream build should still be building with the older golang version so I'd suggest reverting it

Reply:
actually, the lastest go is adding this IMO, before my commit one the ci was failing with 
```
hack/verify-generated.sh
diff --git a/api/v1/zz_generated.deepcopy.go b/api/v1/zz_generated.deepcopy.go
index 26525244..d460b37b 100644
--- a/api/v1/zz_generated.deepcopy.go
+++ b/api/v1/zz_generated.deepcopy.go
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
// +build !ignore_autogenerated

/*
uncommitted generated files. run 'make update-generated' and commit results.
make: *** [Makefile:180: verify-generated] Error 1
```